packages/aws-cdk-lib/aws-cloudwatch/lib/alarm.ts

```diff
@@ -137,6 +163,16 @@ export class Alarm extends AlarmBase {
     return new Import(scope, id);
   }
 
+  /**
+   * Determine whether this operator is an anomaly detection operator.
+   *
+   * @param operator the comparison operator for the alarm.
+   * @returns true if the operator is an anomaly detection operator, false otherwise.
+   */
+  public static isAnomalyDetectionOperator(operator?: ComparisonOperator): boolean {
+    return operator !== undefined && ANOMALY_DETECTION_OPERATORS.includes(operator);
```

Comment:
For my understanding, existing alarms can't use one of the new operator?

Reply:
Here is my understanding, correct me if I'm wrong @scorbiere:

* These operators only make sense in combination with an `ANOMALY_DETECTION_BAND()` function
* Existing Alarms are a superset of Anomaly Detection alarms.

So, if a user puts an `ANOMALY_DETECTION_BAND` function into their metric they can use the operator with the old Alarm, otherwise not.

Just by the fact that the user is using `Alarm` we can't tell whether the operator is correct or not, but we
could conceivably combine it with some sort of validation on the presence of the right function in `Alarm`.

```ts
if (usesAnomalyFunction(props.metric) !== isAnomlyDetectionOperator(props.operator)) {
  throw new Error('oh no');
}
```